frontend/src/components/news-page/NewsCard.scss

```diff
@@ -0,0 +1,11 @@
+// .video-container {
+//   width: 100%;
+//   aspect-ratio: 1 / 1;
+//   overflow: hidden;
+//   display: block;
+// }
+
+video {
+  width: 100% !important;
```

Comment:
`!important` is generally (but not always) wise to avoid. Is it definitely necessary here?